zephyr/ra/ra_cfg/fsp_cfg/bsp/ra2a1/bsp_clock_cfg.h

```diff
@@ -0,0 +1,129 @@
+#define BSP_CFG_HOCO_FREQUENCY 2 /* HOCO 32MHz */
+#elif DT_PROP(DT_NODELABEL(hoco), clock_frequency) == 48000000
+#define BSP_CFG_HOCO_FREQUENCY 4 /* HOCO 48MHz */
+#elif DT_PROP(DT_NODELABEL(hoco), clock_frequency) == 64000000
+#define BSP_CFG_HOCO_FREQUENCY 5 /* HOCO 64MHz */
+#else
+#error "Invalid HOCO frequency, only can be set to 24MHz, 32MHz, 48MHz, 64MHz"
+#endif
+
+#define BSP_CFG_PLL_SOURCE                                                                         \
+	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), source, RA_PLL_SOURCE_DISABLE)
+#define BSP_CFG_PLL_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), div, RA_PLL_DIV_1)
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(pll), okay)
+#define BSP_CFG_PLL_MUL                                                                            \
+	BSP_CLOCKS_PLL_MUL(DT_PROP_BY_IDX(DT_NODELABEL(pll), mul, 0),                              \
+			   DT_PROP_BY_IDX(DT_NODELABEL(pll), mul, 1))
+#else
+#define BSP_CFG_PLL_MUL BSP_CLOCKS_PLL_MUL(0, 0)
+#endif
+
+#define BSP_CFG_PLODIVP            BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), divp, RA_PLL_DIV_2)
+#define BSP_CFG_PLL1P_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), freqp, 0)
+#define BSP_CFG_PLODIVQ            BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), divq, RA_PLL_DIV_2)
+#define BSP_CFG_PLL1Q_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), freqq, 0)
+#define BSP_CFG_PLODIVR            BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), divr, RA_PLL_DIV_2)
+#define BSP_CFG_PLL1R_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll), freqr, 0)
+
+#define BSP_CFG_PLL2_SOURCE                                                                        \
+	BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), source, RA_PLL_SOURCE_DISABLE)
+#define BSP_CFG_PLL2_DIV BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), div, RA_PLL_DIV_1)
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(pll2), okay)
+#define BSP_CFG_PLL2_MUL                                                                           \
+	BSP_CLOCKS_PLL_MUL(DT_PROP_BY_IDX(DT_NODELABEL(pll2), mul, 0),                             \
+			   DT_PROP_BY_IDX(DT_NODELABEL(pll2), mul, 1))
+#else
+#define BSP_CFG_PLL2_MUL BSP_CLOCKS_PLL_MUL(0, 0)
+#endif
+
+#define BSP_CFG_PL2ODIVP           BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), divp, RA_PLL_DIV_2)
+#define BSP_CFG_PLL2P_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), freqp, 0)
+#define BSP_CFG_PL2ODIVQ           BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), divq, RA_PLL_DIV_2)
+#define BSP_CFG_PLL2Q_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), freqq, 0)
+#define BSP_CFG_PL2ODIVR           BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), divr, RA_PLL_DIV_2)
+#define BSP_CFG_PLL2R_FREQUENCY_HZ BSP_CLOCK_PROP_HAS_STATUS_OKAY_OR(DT_NODELABEL(pll2), freqr, 0)
```

Comment:
Hello @soburi , RA2A1 didn't support for these clocks. Could you please update them match with supported clock which described in HWM?
Please refer to https://github.com/zephyrproject-rtos/hal_renesas/blob/c2d795cfa138c85d928b69d259897279bae7b9da/zephyr/ra/ra_cfg/fsp_cfg/bsp/ra2a1/bsp_clock_cfg.h for what I mean.

Reply:
Same as above.